canonical/ubuntu.com
# Change references to Ubuntu Advantage to Ubuntu Pro in legal pages

## Done
Change references to Ubuntu Advantage to Ubuntu Pro in legal pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- Browse `/legal` and make sure everything refers to Ubuntu Pro


Fixes https://warthogs.atlassian.net/browse/WD-1483
